multinet/db.py

```diff
@@ -222,13 +229,38 @@ def get_workspace_metadata(name: str) -> Workspace:
 
     # Find the metadata record for the named workspace. If it's not there,
     # something went very wrong, so bail out.
-    metadata = workspace_mapping(name)
+    metadata = workspace_mapping.__wrapped__(name)
     if metadata is None:
         raise DatabaseCorrupted()
 
     return metadata
 
 
+def set_workspace_permissions(
+    name: str, permissions: WorkspacePermissions
+) -> WorkspacePermissions:
+    """Update the permissions for a given workspace."""
+    if not workspace_exists(name):
+        raise WorkspaceNotFound(name)
+
+    doc = copy.deepcopy(workspace_mapping(name))
+    if doc is None:
+        raise DatabaseCorrupted()
+
+    # TODO: Do user object validation once ORM is implemented
```

Comment:
Can you make sure to file an issue for this TODO?

Reply:
https://github.com/multinet-app/multinet-server/issues/444